### PR TITLE
feat: use supermajority agreement + increase elder size to 7

### LINKED ISSUE #2313: Change quorum to supermajority (> 2/3) and increase elder size to 7
The supermajority is necessary for some aspects of the network like AT2, BRB or fork detection. 7 elders is so we can still lose 2 and be OK.

### PR DESCRIPTION
**Pull request by madadam**
Closes #2313

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -105,25 +105,40 @@ mod section;
 pub const RECOMMENDED_SECTION_SIZE: usize = 10;
```
Comment (bochaco) [inline src/lib.rs:105]: Perhaps we need to increase this as well now?

Comment (madadam) [reply src/lib.rs:105]: Good point. What we be a good value? Maybe `2 * ELDER_SIZE` ?

Comment (bochaco) [reply src/lib.rs:105]: yeh perhaps it's better to make it a factor of ELDER_SIZE so we don't leave one of them unchanged when one of them changes.

**Review by Yoga07 — DISMISSED** [review]

**Review by joshuef — APPROVED** [review]
Works w/ node amster and client master 👍

**Pull request merged**